packages/desktop-client/src/components/reports/ReportCard.tsx

```diff
@@ -1,11 +1,26 @@
-// @ts-strict-ignore
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏nice!